pydicom/tests/test_charset.py

```diff
@@ -11,16 +11,16 @@
 import os.path
 import pydicom.charset
 
-test_dir = os.path.dirname(__file__)
-testcharset_dir = os.path.join(test_dir, 'charset_files')
+test_dir, _ = os.path.split(os.path.dirname(__file__))
```

Comment:
Can we move all of these directory calculations into a central place so we don't have to repeat this for every separate test file?